src/org/jetbrains/java/decompiler/api/plugin/LanguageSpec.java

```diff
@@ -8,12 +8,18 @@ public final class LanguageSpec {
   public final GraphParser graphParser;
   public final StatementWriter writer;
   public final Pass pass;
-
+  public final String extension;
+  
   public LanguageSpec(String name, LanguageChooser chooser, GraphParser graphParser, StatementWriter writer, Pass pass) {
+    this(name, chooser, graphParser, writer, pass, "java");
```

Comment:
I think this should be removed. It's technically a breaking change but I don't think anyone else has written language plugins yet and defaulting to java is not ideal.